front-end/src/timers/target.jsx

```diff
@@ -182,13 +169,13 @@ const Target = (props) => {
             <label htmlFor='message'>{i18next.t('timers:message')}</label>
             <Field
               component='textarea'
-              name={NameFor(name, 'message')}
-              disabled={readOnly}
+              name={NameFor(props.name, 'message')}
+              disabled={props.readOnly}
               className={classNames('form-control', {
-                'is-invalid': ShowError(NameFor(name, 'message'), touched, errors)
+                'is-invalid': ShowError(NameFor(props.name, 'message'), props.touched, props.errors)
               })}
             />
-            <ErrorFor {...props} name={NameFor(name, 'message')} />
+            <ErrorFor {...props} name={NameFor(props.name, 'message')} />
```

Comment:
i was close :-) , but still dont know what im doing

Reply:
Ah.  That probably wasn't necessary.  I can change back to using let if you would like. Root cause was that Formik didn't know the shape of the data.